aea/configurations/base.py

```diff
@@ -2330,28 +2331,38 @@ def _check_aea_version(package_configuration: PackageConfiguration):
 
 
 def _compare_public_ids(
-    package_configuration: PackageConfiguration, package_directory: Path
+    component_configuration: ComponentConfiguration, package_directory: Path
```

Comment:
I use `ComponentConfiguration` to use `prefix_import_path`, although that is not strictly needed